src/lib/log.js

```diff
@@ -244,6 +244,25 @@ in a later version. Use \`${newUsage}\` instead`);
       this._logImageInNode({image, message, scale});
   }
 
+  // Logs the current settings as a table
+  status() {
+    if(console.table) {
```

Comment:
Not sure this if statement is needed, I think there is a polyfill for console.table somewhere in probe.gl?